src/libs/antares/study/parts/short-term-storage/container.cpp

```diff
@@ -155,13 +160,19 @@ bool STStorageInput::loadAdditionalConstraints(const fs::path& parentPath)
             }
         }
 
-        if (const auto rhsPath = parentPath / (
-                                     "rhs_" + additionalConstraints.name
-                                     + ".txt"); !readRHS(additionalConstraints, rhsPath))
+        // We don't want load RHS and link the STS time if the constraint is disabled
+        if (!additionalConstraints.enabled)
+        {
+            return true;
+        }
```

Comment:
If the constraint is disabled, we don't load the timeseries and link the STS object.